### PR TITLE
Upgrade platform integration to use CLIv2

### PR DESCRIPTION
**Pull request by michaelloosen**
To keep in sync with Platform enhancements and to allow the ease of deploying JeMPI-web locally.

### CODE REVIEW

**Comment by marrouchi** [comment]
Hello @michaelloosen , If I understand correctly, if get-cli.sh or swarm.sh gets updated in platform, it needs to be updated here as well. I think we need to come up of a better way to handle sync. I will try to think of something we could do.

In the mean time maybe we should use github auto sync workflow for now : https://medium.com/swlh/how-to-auto-sync-update-from-one-github-repository-to-other-repository-using-github-workflow-6492f4830d45

**Comment by marrouchi** [comment]
It seems there's a read-to-use sync action : https://github.com/marketplace/actions/repo-file-sync-action

File: .env.local [hunk 1]
```diff
@@ -1,3 +1,5 @@
+CLUSTERED_MODE=false
```
Comment (nour-borgi) [inline .env.local:1]: There is a variable "CLUSTERED" in this file, I think we can remove it like in .env.test

**Review by nour-borgi — APPROVED** [review]
looks good just one change

File: .env.local (hunk 1 above)
Comment (michaelloosen) [reply .env.local:1]: Thanks, missed that 👍

**Pull request merged**